currencyConverter.js

```diff
@@ -0,0 +1,66 @@
+var request = require('request');
+var cheerio = require('cheerio');
+
+var roundOff = function(currency){
+
+	/**
+	* Rounds off currency output 2 decimal places
+	*/
+
+	return Math.round(currency * 100) / 100;
+
+}
+
+var getCurrency = function(param, param1){
+
+	/**
+	* Converts two parameters from param to param1 eg: USD to ZAR
+	* @param string selects the currency you would like to convert
```

Comment:
In JSDoc, you need to specify the parameter name, so here it will be:

@param param
@param param1

Obviously different when you change those names.